qa/L0_backend_python/io/io_test.py

```diff
@@ -101,57 +100,56 @@ def _run_ensemble_test(self):
                             self.assertTrue(np.all(output0 == input0))
 
     def test_ensemble_io(self):
-        # Only run the shared memory leak detection with the default trial
```

Comment:
This was previously skipping decoupled trial - trying to add decoupled test back.